docs/api.rst

```diff
@@ -1178,3 +1181,1395 @@ The brokers resource provides access to the current state of Kafka brokers in th
+
+   :<jsonarr partitions: A list of partitions to assign to this consumer
+   :<jsonarr string partitions[i].topic: Name of the topic
+   :<jsonarr int partitions[i].partition: Partition ID
+ 
+   :statuscode 404:
+      * Error code 40403 -- Consumer instance not found
+	
+   :statuscode 409:
+      * Error code 40903 -- Subscription to topics, partitions and pattern are mutually exclusive.
+	
+   :statuscode 500:
+      * Error code 500 -- General consumer error response, caused by an exception during the
+        operation. An error message is included in the standard format which explains the cause.
+
+
+   **Example request**:
+
+   .. sourcecode:: http
+
+      POST /consumers/testgroup/instances/my_consumer/assignments HTTP/1.1
+      Host: proxy-instance.kafkaproxy.example.com
+      Content-Type: application/vnd.kafka.v2+json      
+
+      {
+        "partitions": [
+          {
+            "topic": "test",
+            "partition": 0
+          },
+          {
+            "topic": "test",
+            "partition": 1
+          }
+	  
+        ]
+      }
+      
+   **Example response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 204 No Content
+
+
+
+.. http:get:: /consumers/(string:group_name)/instances/(string:instance)/assignments
+	       
+   Get the list of partitions currently manually assigned to this consumer.
+   
+   :param string group_name: The name of the consumer group
+   :param string instance: The ID of the consumer instance
+
+   :>jsonarr partitions: A list of partitions manually to assign to this consumer
+   :>jsonarr string partitions[i].topic: Name of the topic
+   :>jsonarr int partitions[i].partition: Partition ID
+
+   :statuscode 404:
+      * Error code 40403 -- Consumer instance not found
+   :statuscode 500:
+      * Error code 500 -- General consumer error response, caused by an exception during the
+        operation. An error message is included in the standard format which explains the cause.
+
+   **Example request**:
+
+   .. sourcecode:: http
+
+      GET /consumers/testgroup/instances/my_consumer/assignments HTTP/1.1
+      Host: proxy-instance.kafkaproxy.example.com
+      Accept: application/vnd.kafka.v2+json      
+
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/vnd.kafka.v2+json
+
+      {
+        "partitions": [
+          {
+            "topic": "test",
+            "partition": 0
+          },
+          {
+            "topic": "test",
+            "partition": 1
+          }
+	  
+        ]
+      }
+
+
+.. http:post:: /consumers/(string:group_name)/instances/(string:instance)/positions
+
+   Overrides the fetch offsets that the consumer will use for the next set of records to fetch.
+
+   :param string group_name: The name of the consumer group
+   :param string instance: The ID of the consumer instance
+
+   :<jsonarr offsets: A list of offsets
+   :<jsonarr string offsets[i].topic: Name of the topic for 
```

Comment:
sentence fragment. maybe just "Name of the topic", or if there was supposed to be more then "Name of the topic for the offset to seek to" or something like that?